pkgs/os-specific/linux/kbd/default.nix

```diff
@@ -57,16 +39,16 @@ stdenv.mkDerivation rec {
     done
   '';
 
-
   buildInputs = [ check pam ];
   nativeBuildInputs = [ autoreconfHook pkg-config flex ];
 
-  makeFlags = [ "setowner=" ];
```

Comment:
I just removed the line `makeFlags = [ "setowner=" ];` because it seem to work without it. Is there any reason why it might be needed?

cc @abbradar who added it in https://github.com/NixOS/nixpkgs/commit/853018124668d83bc17815da52d0e68b1a163eb7

That's the last bit where i'm unsure, the rest of the PR seems good to me now.

Reply:
It wasn't @abbradar but @edolstra in 65e004b75775fa4e3d5f9a40be92e0655774b005.

Nevertheless, there is no ocurrence of `setowner` anywhere in the source code, so I think it's fine to drop this.